Source/UnitTests/VideoCommon/VertexLoaderTest.cpp

```diff
@@ -50,8 +48,12 @@ class VertexLoaderTest : public testing::Test
     memset(input_memory, 0, sizeof(input_memory));
     memset(output_memory, 0xFF, sizeof(input_memory));
 
-    memset(&m_vtx_desc, 0, sizeof(m_vtx_desc));
-    memset(&m_vtx_attr, 0, sizeof(m_vtx_attr));
+    m_vtx_desc.low.Hex = 0;
+    m_vtx_desc.high.Hex = 0;
+
+    m_vtx_attr.g0.Hex = 0;
```

Comment:
This can be shortened to:

```suggestion
    m_vtx_attr = {};
```

Reply:
This doesn't work because the copy assignment operator for BitField has been deleted.  

I tried creating a copy assignment operator for TVtxDesc and VAT to copy the respective member .Hex fields, but kept getting errors (I believe from the BitFieldArrays in TVtxDesc .low and .high) about the operation not being sane.